yarn.lock

```diff
@@ -8428,6 +8447,11 @@ dom-accessibility-api@^0.5.6:
   resolved "https://registry.yarnpkg.com/dom-accessibility-api/-/dom-accessibility-api-0.5.6.tgz#3f5d43b52c7a3bd68b5fb63fa47b4e4c1fdf65a9"
   integrity sha512-DplGLZd8L1lN64jlT27N9TVSESFR5STaEJvX+thCby7fuCHonfPpAlodYc3vuUYbDuDec5w8AMP7oCM5TWFsqw==
 
+dom-accessibility-api@^0.5.9:
```

Comment:
This can be coalesced with the dep above it. You can do it manually by deleting lines 8445-8449 and changing 8450 to

    dom-accessibility-api@^0.5.6, dom-accessibility-api@^0.5.9:

You can also do it via CLI with `npx yarn-deduplicate --packages dom-accesibility-api`

Reply:
done ty